ingress/controllers/nginx/nginx/command.go

```diff
@@ -29,6 +29,14 @@ import (
 
 	"k8s.io/contrib/ingress/controllers/nginx/nginx/config"
 	"k8s.io/contrib/ingress/controllers/nginx/nginx/ingress"
+
+	"expvar"
+)
+
+// number of reload performed
+var (
+	reload = expvar.NewInt("reload")
```

Comment:
well, this is a way to create a variable for expvar library.
It will then be exposed to this [path](https://github.com/kubernetes/contrib/pull/1835/files#diff-c5f50015a7af6b5dfb050056b4d9338fR165) though the http.DefaultServeMux